app/controllers/psychologist/registrations_controller.rb

```diff
@@ -7,13 +7,26 @@ class Psychologist::RegistrationsController < Devise::RegistrationsController
   def new
     super
   end
+   
+  def update
+    super
+  end
+
+  protected
+
+  def after_update_path_for(profile_psychologist)
```

Comment:
@xFerchoVJ  probably you could call this method using the `after_action` [Filters](https://guides.rubyonrails.org/action_controller_overview.html#filters)